slsim/Util/astro_util.py

```diff
@@ -555,6 +559,8 @@ def generate_signal(
     high_frequency_slope=3,
     new_mean_amplitude=0,
     new_standard_deviation=1,
+    is_magnitude=False,
```

Comment:
Can you make default is_magnitude = True because magnitude is the conventional unit in SLSim?